intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/YangGrammar.bnf

```diff
@@ -249,7 +249,7 @@ typedef-stmt-body ::= (type-stmt stmtsep) |
 type-stmt ::= TYPE_KEYWORD sep identifier-ref-arg-quoted optsep
  ( SEMICOLON |
  ( LEFT_BRACE stmtsep
- type-body-stmts + // + is not corresponding with rfc6020 (added additionally)
+ type-body-stmts // + is not corresponding with rfc6020 (added additionally)
```

Comment:
this statement is now corresponding with rfc so there is no need for that comment

maybe change comment that there used to be + before if reason why i added it comes up again :) i wish i did comment also reason for that change because now only got knows knows reasons for that change :)

Reply:
![254630560_5096234513723219_8380663400061133446_n](https://user-images.githubusercontent.com/20628428/140738048-b69ecc03-9b2c-4a59-aba0-faa6fdd362f7.jpg)